tutorial_forecast_as_service/api/services.py

```diff
@@ -0,0 +1,50 @@
+"""
+Import as:
+
+import tutorial_forecast_as_service.api.services as tfasapse
+"""
+
+import io
+import logging
+import os
+from typing import Any, Dict, List
+
+import fastapi
+import pandas as pd
+
+import tutorial_prophet.src.prophet_model as tpsrprmo
+
+_LOG = logging.getLogger(__name__)
+
+_UPLOAD_PATH = "tmp/uploaded_df.pkl"
+
+
+def handle_upload(file: fastapi.UploadFile) -> Dict[str, str]:
+    """
+    Read and parse uploaded CSV file, then persist to disk.
+
+    :param file: uploaded file object from FastAPI
+    :return: upload confirmation message
+    """
+    contents = file.file.read()
+    df = pd.read_csv(io.BytesIO(contents))
+    _LOG.info("Data uploaded with shape: %s", df.shape)
+    os.makedirs("tmp", exist_ok=True)
```

Comment:
ok, but there is a hio.create_dir that has more functions

Also let's call the dir `tmp.forecast_as_service.upload` so we know what it is